packages/thrift-server-core/src/main/protocols/CompactProtocol.ts

```diff
@@ -790,7 +790,7 @@ export class CompactProtocol extends TProtocol {
         if (!isInt64(i64)) {
             throw new TProtocolException(
                 TProtocolExceptionType.INVALID_DATA,
-                `Expected Int64 or Number, found: ${i64}`,
```

Comment:
This is a slightly different error type but I think logging the type rather than the stringified value makes sense here still.

Reply:
I don't see a pattern as to why this error would be different (maybe it shouldn't be) but I didn't change it.